spec/lib/parser_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.describe Parser do
+  describe '#klass' do
+    subject { described_class.new(raw_data).klass }
+
+    describe 'returns correct class for rails 3' do
+      let(:raw_data) do
+        { 'meta' => {"client_version"=>"0.0.7", "rails_version"=>"3.2.13"} }
```

Comment:
Surrounding space missing for operator `=>`.<br>Space inside { missing.<br>Space inside } missing.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.